remote-build.exp

```diff
@@ -123,6 +128,13 @@ if { ! [file exists configs/${at_version}] } {
 puts "AT version: $at_version"
 puts "Servers: $servers"
 
+# Define default value of variables if not previously defined
+# Directory used by the script to save its files.
+set_def tray_dir "~/$basedir/tmp/at-build-tray"
+# Directory where AT will be installed (aka DESTDIR).
+set_def destdir "~/$basedir/opt"
```

Comment:
@tuliom Aha, not really! I mean, I think. Apparently for bash "x/./y" and "x//y" mean the same as "x/y", and "." is actually the default value for $basename if the the user doesn't specify --dir :)

Also, I'd like to mention that I did test the behavior I mentioned here, but I'm not sure if it's exactly... "canonical". In case there's a more standard way to do it, please let me know.